verl/utils/checkpoint/fsdp_checkpoint_manager.py

```diff
@@ -342,3 +386,25 @@ def save_checkpoint(self, local_path: str, hdfs_path: str = None, global_step: i
             torch.distributed.barrier()
 
         self.previous_saved_paths.append(local_path)
+
+    def upload_checkpoint_in_background(self, s3_path: str, local_path: str) -> None:
```

Comment:
is this a public API? If not please rename it to `_upload_checkpoint_in_background`.